### PR TITLE
Release 3.10

### PR DESCRIPTION
**Pull request by whimboo**
This PR fixes issue #238. I will still have to wait for PR #237 landed first before merging it.

@xabolcs, can you please have a look?

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+3.10 / 2017-05-16
+==================
+  * Update maxVersions of applications in install.rdf (#310)
```
Comment (xabolcs) [inline History.md]: This is not Issue #310 (, but #237?)

File: extension/install.rdf [hunk 2]
```diff
@@ -65,7 +65,7 @@
       <Description>
         <em:id>{3550f703-e582-4d05-9a08-453d09bdfdc6}</em:id>
         <em:minVersion>39.0</em:minVersion>
-        <em:maxVersion>52.0</em:maxVersion>
+        <em:maxVersion>59.0</em:maxVersion>
```
Comment (xabolcs) [inline extension/install.rdf]: `59.0` is too much. `56.0` or `*` is the currently acceptable `maxVersion` for <del>Firefox</del> Thunderbird on AMO.
We shouldn't pass `56.0` anyway!

File: extension/install.rdf [hunk 3]
```diff
@@ -85,7 +85,7 @@
       <Description>
         <em:id>toolkit@mozilla.org</em:id>
         <em:minVersion>39.0</em:minVersion>
-        <em:maxVersion>52.0</em:maxVersion>
+        <em:maxVersion>59.0</em:maxVersion>
```
Comment (xabolcs) [inline extension/install.rdf]: Same here.

File: extension/install.rdf [hunk 4]
```diff
@@ -56,7 +56,7 @@
       <Description>
         <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
         <em:minVersion>39.0</em:minVersion>
-        <em:maxVersion>52.0</em:maxVersion>
+        <em:maxVersion>59.0</em:maxVersion>
```
Comment (xabolcs) [inline extension/install.rdf]: Same here for Firefox, let it to be `56.0`!

File: extension/install.rdf [hunk 5]
```diff
@@ -74,7 +74,7 @@
       <Description>
         <em:id>{92650c4d-4b8e-4d2a-b7eb-24ecf4f6b63a}</em:id>
         <em:minVersion>2.38</em:minVersion>
-        <em:maxVersion>2.49</em:maxVersion>
+        <em:maxVersion>2.53</em:maxVersion>
```
Comment (xabolcs) [inline extension/install.rdf]: :+1:

**Comment by xabolcs** [comment]
[Application versions](https://addons.mozilla.org/en-US/firefox/pages/appversions/) on AMO, for reference.

File: History.md (hunk 1 above)
Comment (whimboo) [inline History.md]: Ups, not sure why the release version made it as issue #. Thanks for noticing.

File: extension/install.rdf (hunk 2 above)
Comment (whimboo) [inline extension/install.rdf]: Oh, right. That's definitely true. Totally missed that with 57.0 the current extension won't work anymore.

**Comment by whimboo** [comment]
Should be all done now.

**Comment by xabolcs** [comment]
Looks good to me now! :+1:

File: History.md [hunk 6]
```diff
@@ -1,3 +1,10 @@
+3.10 / 2017-05-16
+==================
+  * Update maxVersions of applications in install.rdf (#238)
+  * Revert separator for extensions to new line (#230)
+  * Mark extension as multi-process compatible (#228)
+  * Update ja and ja-JP-mac translation (#233)
+
```
Comment (xabolcs) [inline History.md]: This list should be updated once #240 got merged!

Comment (whimboo) [inline History.md]: We don't have to given that we reference the issue and not the actual PR. So all is still fine.

Comment (whimboo) [inline History.md]: Oh, I will do to have the correct commit message presented there.

**Comment by whimboo** [comment]
PR has been updated for the wording of issue #230.

**Comment by xabolcs** [comment]
Looks good to me now! :+1:

**Pull request merged**